### PR TITLE
add enabled_features to the heartbeat document

### PR DESCRIPTION
**Pull request by ferullo**
## Change Summary

Adds `Endpoint.state.enabled_features` to the heartbeat document. I removed the `custom_documentation` since these are moving to a `.logs` data stream.

### Sample values

See PR content.

## Release Target

8.10

### For mapping changes:

- [x] I ran `make` after making the schema changes, and committed all changes

### CODE REVIEW

File: package/endpoint/data_stream/heartbeat/sample_event.json [hunk 1]
```diff
@@ -11,5 +11,17 @@
     "message": "Endpoint heartbeat",
     "event": {
         "ingested": "2023-07-18T20:40:09.279939Z"
+    },
+    "Endpoint": {
+        "state": {
+            "enabled_features": [
```
Comment (ferullo) [inline package/endpoint/data_stream/heartbeat/sample_event.json:17]: @joeypoon what do you think of this exact location? `Endpoint.state.host_isolation=true|false` already exists. It felt like this belonged along side it conceptually.

**Review by pzl — APPROVED** [review]
looks good to me, assuming the removal from `custom_documentation` was intentional

File: package/endpoint/data_stream/heartbeat/sample_event.json (hunk 1 above)
Comment (joeypoon) [reply package/endpoint/data_stream/heartbeat/sample_event.json:17]: make sense to me 👍

**Review by joeypoon — APPROVED** [review]

**Comment by ferullo** [comment]
I tested this and it works with upcoming Endpoint changes to add this data.

**Pull request merged**